RecogScript.py

```diff
@@ -18,17 +18,14 @@
 def process_video_to_encode(video_feed):
 
     all_ids, all_encodings = get_all_people_information()
-
     frame = get_frame(video_feed)
-
     frame_encodings = get_face_encodings(frame)
-
     encodings = compare_encodings(frame_encodings, all_encodings)
 
     return encodings, all_ids, frame 
 
 
-def check_encodings(all_encodings, all_ids, small_frame, temp_filename="images/temp.jpeg"):
+def check_encodings(all_encodings, all_ids, small_frame, temp_filename):
```

Comment:
this is fine but basically the old line means that if we forget to supply a temp_filename parameter, then images/temp.jpeg would be used by default... I see why you got rid of it but a heads up

Reply:
sounds good